heron/api/tests/scala/com/twitter/heron/streamlet/scala/BuilderTest.scala

```diff
@@ -0,0 +1,47 @@
+//  Copyright 2018 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.streamlet.scala
+
+import scala.collection.mutable.ListBuffer
+import com.twitter.heron.streamlet.Context
+import com.twitter.heron.streamlet.scala.common.{BaseFunSuite, TestContext}
+
+
+/**
+  * Tests for Builder Trait functionality
+  */
+class BuilderTest extends BaseFunSuite {
+
+
+  test("Builder should provide new source based on user defined supplier function") {
+    assert(1==1)
```

Comment:
??

Reply:
I removed this class as its no longer needed